src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/custom.py

```diff
@@ -2,3 +2,27 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+
+
+def list_metric_definitions(client, resource_uri, metric_names=None):
+    '''Commands to manage metric definitions.
+    :param str resource_uri: The identifier of the resource
+    :param str metric_names: The list of metric names
+    '''
+    odata_filter = _list_metric_definitions_filter_builder(metric_names)
+    metric_definitions = client.list(resource_uri, filter=odata_filter)
+    return list(metric_definitions)
+
+
+def _list_metric_definitions_filter_builder(metric_names=None):
+    '''Build up OData filter string from metric_names
+    '''
+    filters = []
+    if metric_names:
+        if "," in metric_names:
```

Comment:
Generally in the CLI we take space separated values for 'lists of things' instead of separating with a comma.
So
`--metric-names "AverageResponseTime,AverageMemoryWorkingSet"`
would become
`--metric-names  AverageResponseTime AverageMemoryWorkingSet `

Then in params, the param is defined  with `nargs='+'`.

@tjprescott do you agree?